### PR TITLE
Rename Driver into InteractionDriver + introduce RBInstanceVariableHa…

### PR DESCRIPTION
**Pull request by Ducasse**
…sReferences

### CODE REVIEW

File: src/Kernel/BlockClosure.class.st
Comment (balsa-sarenac) [inline src/Kernel/BlockClosure.class.st:1]: Was this removed on purpose? Seemed kind of strange, so just wanted to double-check.

File: src/Refactoring-Tests-Core/RBConditionTest.class.st [hunk 1]
```diff
@@ -122,13 +122,26 @@ RBConditionTest >> testHierarchyOfReferencesInstanceVariable [
 
 	| testClass |
 	testClass := rbNamespace classNamed: self class name.
-
+	self halt.
```
Comment (balsa-sarenac) [inline src/Refactoring-Tests-Core/RBConditionTest.class.st]: This halt was left here, not sure if it is on purpose?

**Review by balsa-sarenac — COMMENTED** [review]
This looks super nice! Left two minor comments

File: src/Kernel/BlockClosure.class.st
Comment (Ducasse) [reply src/Kernel/BlockClosure.class.st:1]: No this is super strange, I have no idea why. And in addition I removed it from the merge. :(
I have no idea how to fix it so may be I will recommit

File: src/Refactoring-Tests-Core/RBConditionTest.class.st (hunk 1 above)
Comment (Ducasse) [reply src/Refactoring-Tests-Core/RBConditionTest.class.st]: I will fix it too.

File: src/Kernel/BlockClosure.class.st
Comment (jecisc) [reply src/Kernel/BlockClosure.class.st:1]: I fixed it for you

Comment (Ducasse) [reply src/Kernel/BlockClosure.class.st:1]: Thanks! I was looking for it and I could not find!!!!

Comment (jecisc) [reply src/Kernel/BlockClosure.class.st:1]: What I did was to be outside Pharo, revert your commit but then I just commited the reversal of this file and droped the other changes of this revert

**Pull request merged**